hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson

```diff
@@ -43,15 +43,22 @@
     }
     {
       name: tl_d_partial_access
-      desc: '''Access CSR with one or more bytes of data
-            For read, expect to return all word value of the CSR
-            For write, enabling bytes should cover all CSR valid fields'''
+      desc: '''Access CSR with one or more bytes of data.
+            For read, expect to return all word value of the CSR.
+            For write, enabling bytes should cover all CSR valid fields.'''
       milestone: V2
       tests: ["{name}_csr_hw_reset",
               "{name}_csr_rw",
               "{name}_csr_aliasing",
               "{name}_same_csr_outstanding"]
     }
+    {
+      name: tl_integrity_check
+      desc: '''Backdoor inject ECC errors during CSR accesses, and verify the correct fatal alert
+            is triggered.'''
```

Comment:
```suggestion
      desc: '''Verify that the data integrity check violation generates an alert.
      
            Randomly inject errors on the control, data or the ECC bits during CSR accesses.
            Verify that triggers the correct fatal alert is triggered.'''
```

Recommend writing the description as a short one-liner, followed by a new paragraph explaining the stimulus / checks.

Reply:
Got it, thanks Sri for the detailed suggestion.